components/docs-chef-io/content/automate/servicenow_integration.md

```diff
@@ -0,0 +1,375 @@
+
+#### Transform Map Scripts
+
+The **Transformation** events occur during the process of transforming an import set table in to a target table. You can navigate to this script by selecting **Importing Data** >**Transform Event Scripts**.
+
+- **on Before:** Mapping manufacturer, CPU manufacturer, model category, and model by using this script.
+
+- **on After:** 
+
+  - First on after script is used to update **node id**, **node name**, **organization**, **IP address** and insert the record into Chef Automate client run cookbooks, Chef Automate client run recipes, Chef Automate client run lists.
+  - Second on after script is used to update **software**, **name** and installed on in _CMDB software instance_ table.
+  - The third one is used to update data in the _cmdb\_ci\_spkg_ table and the _cmdb\_software\_instance_ table.
+  - The fourth one is used to insert or update data in the _cmdb\_ci\_file\_system_ table.
+  - The fifth one is used to insert or update data in the _cmdb\_ci\_network\_adapter_ table. 
+
+{{< figure src="/images/automate/snow_integration_transform_map_scripts.png" alt="Transform Scripts">}}
+
+##### *Script Includes:*
+
+- **BufferToImportUtil:** Updates the CPU speed in Linux server, CMDB server, CMDI OSX server, and CMDB win server.
+- **Logging:** Updates logging status as **enabled** or **disabled**.
+- **JsonUtil:** Updates the JSON objects.
+- **AutomateApi:** Calls the Chef automate API and checks the status of the response.
+- **AutomateApiClient:** Updates the token and password in the Chef Automate instance.
+- **PropertiesUtil:** Updates the role in the property.
+- **ImportUtil:** Updates the compliance import data.
+- **Util:** Updates manufacturer data in the core company table, and updates the category and manufacturer data in the _CMDB model_ table.
+
+{{< figure src="/images/automate/snow_integration_transform_map_scripts1.png" alt="Script Includes">}}
+
+## Configuration
+
+Chef Automate can be configured in two ways:
+
+- Using the ServiceNow application
+- Using the Chef Automate UI
+
+**Precaution** - It is recommended to take a cautionary approach to set up the data imports using Chef Automate's **Data Feed** feature to avoid unnecessary performance impacts to your ServiceNow infrastructure. In addition, it is recommended that larger time intervals for the **feed_interval** setting, such as **4 or 8 hours**, are used initially and with a smaller **node_batch_size** of **50**. It is adviced to test it on your development infrastructure by making use of the **CIDR Filter** functionality. The Chef Automate's **Data Feed** feature may suit your production environment, depending on the size of your estate and the set up you made in the ServiceNow production environment.
+
+### Configuring Chef Automate from ServiceNow
+
+The configuration of a Chef Automate from a ServiceNow production instance requires the Chef Automate instance to be configured with a valid signed SSL certificate.
```

Comment:
This sentence is confusing (` a Chef Automate from a ServiceNow production instance`) and passive (`to be configured`)

Reply:
made a minor change.. I guess, its clear now. Please recheck.